src/index.js

```diff
@@ -367,6 +442,11 @@ export default class AuthnWidget {
             disabled = true;
           }
         }
+        if (input.type === 'text' && input.id === 'otp') {
```

Comment:
Is this configurable somewhere? I would worry about hard coding this if that is the case

Reply:
I suppose if you change the handlebar element for the input as that is what the input type and id correspond to. Is there a better way I should be doing this?

Reply:
Doh! Sorry looks like I got the wrong line # with this one

I meant the OTP length being configurable. Seems like something we would typically allow customers to customize if need be. If there is an option to configure, then I think we should just treat any length as valid

Reply:
no, the otp length is not configurable unfortunately, only a length of 6. The max length a user can enter into the input element on the handlebar is also set to 6.

Reply:
Gotcha, sounds good then. Thanks for the info